bitnami/fluentd/templates/aggregator-statefulset.yaml

```diff
@@ -96,6 +96,11 @@ spec:
               cp -r /opt/bitnami/fluentd/extensions/* /extensions/
               cp -r /opt/bitnami/fluentd/gems/* /gems/
               cp -r /opt/bitnami/fluentd/specifications/* /specifications/
+          {{- if .Values.initContainers.installExtraGems.resources }}
+          resources: {{- toYaml .Values.initContainers.installExtraGems.resources | nindent 12 }}
+          {{- else if ne .Values.initContainers.installExtraGems.resourcesPreset "none" }}
+          resources: {{- include "common.resources.preset" (dict "type" .Values.initContainers.installExtraGems.resourcesPreset) | nindent 12 }}
+          {{- end }}
```

Comment:
We can use the main container resource parameters (`.Values.aggregator.resources` and `.Values.aggregator.resourcePresets`) instead. However, in case you consider it's still necessary to customize the init-container resources (since the main container ones are over-scaled for init-containers) let's use a new parameters (e.g. `.Values.aggregator.initResources` & `.Values.aggregator.initResourcePresets`) but let's try not to modify the `.Values.initContainers` parameter to avoid breaking backwards compatibility.